.github/workflows/continuous-integration.yml

```diff
@@ -156,3 +206,337 @@ jobs:
+            custom-entrypoint: ''
+          - mysql-version: '8.0'
+            # https://stackoverflow.com/questions/60902904/how-to-pass-mysql-native-password-to-mysql-service-in-github-actions
+            custom-entrypoint: >-
+              --entrypoint sh mysql:8 -c "exec docker-entrypoint.sh mysqld --default-authentication-plugin=mysql_native_password"
+          - config-file: "mysql-tls"
+            php-version: "7.4"
+            mysql-version: "8.0"
+            driver: "mysqli"
+
+    services:
+      mysql:
+        image: mysql:${{ matrix.mysql-version }}
+
+        options: >-
+          --health-cmd "mysqladmin ping --silent"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+          -e MYSQL_ALLOW_EMPTY_PASSWORD=yes
+          -e MYSQL_DATABASE=doctrine_tests
+          ${{ matrix.custom-entrypoint }}
+
+        ports:
+          - "33306:3306"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+        with:
+          fetch-depth: 2
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          php-version: "${{ matrix.php-version }}"
+          coverage: "pcov"
+
+      - name: "Cache dependencies installed with composer"
+        uses: "actions/cache@v2"
+        with:
+          path: "~/.composer/cache"
+          key: "php-${{ matrix.php-version }}-composer-locked-${{ hashFiles('composer.lock') }}"
+          restore-keys: "php-${{ matrix.php-version }}-composer-locked-"
+
+      - name: "Install dependencies with composer"
+        run: "composer install --no-interaction --no-progress --no-suggest"
+
+      - name: "Copy TLS-related files"
+        run: 'docker cp "${{ job.services.mysql.id }}:/var/lib/mysql/ca.pem" . && docker cp "${{ job.services.mysql.id }}:/var/lib/mysql/client-cert.pem" . && docker cp "${{ job.services.mysql.id }}:/var/lib/mysql/client-key.pem" .'
+        if: ${{ endsWith(matrix.config-file, 'tls') }}
+
+      - name: "Run PHPUnit"
+        run: "vendor/bin/phpunit -c ci/github/phpunit.${{ matrix.config-file }}.${{ matrix.driver }}.xml --coverage-clover=coverage.xml"
+
+      - name: "Upload Code Coverage"
+        uses: "codecov/codecov-action@v1"
+
+  phpunit-mssql:
+    name: "PHPUnit with MSSQL"
+    runs-on: "ubuntu-latest"
+    needs: phpunit-sqlite
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.3"
+          - "7.4"
+        driver:
+          - "sqlsrv"
+          - "pdo_sqlsrv"
+        collation:
+          - Latin1_General_100_CI_AS
+        include:
+          - collation: Latin1_General_100_CS_AS
+            php-version: "7.4"
+            driver: "sqlsrv"
+          - collation: Latin1_General_100_CS_AS
+            php-version: "7.4"
+            driver: "pdo_sqlsrv"
+
+    services:
+      mssql:
+        image: microsoft/mssql-server-linux:2017-latest
+        env:
+          ACCEPT_EULA: 'Y'
+          SA_PASSWORD: 'Doctrine2018'
```

Comment:
Any reason to use single quotes here? I believe we should use quotes only where necessary, e.g. in the strings like `"7.4"` or `"3306:3306"`.

Reply:
If you want to use quotes consistently, let's then use double quotes by default.